src/anm/player.js

```diff
@@ -380,6 +380,7 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
                         player.state.happens = C.LOADING;
                         player.fire(C.S_CHANGE_STATE, C.LOADING);
                         player.fire(C.S_LOAD, result);
+                        player._updateAudioVolumes();
```

Comment:
maybe, but this method is used in `.volume()` anyway, so it would just complicate things